src/utils/crvoting-utils.js

```diff
@@ -182,10 +182,13 @@ export function removeCodeFromLocalStorage(connectedAccount, disputeId) {
 }
 
 // Auto reveal service preference
-export function getAutoRevealPreference(connectedAccount) {
-  return Boolean(localStorage.getItem(`autoRevealService:${connectedAccount}`))
+export function getAutoRevealPreference(connectedAccount, voteId) {
+  return localStorage.getItem(`autoRevealService:${connectedAccount}:${voteId}`)
```

Comment:
Ah yeah, the `Boolean` conversion wasn't accurate because we are storing the Boolean type which is converted to string, so if the auto reveal is `flase` gave a wrong result. Though i will move the updated evaluation inside this function again.